### PR TITLE
contributing.md

### LINKED ISSUE #60: Update the contributing.md with details around changes and releases
We need to document our ways of working with respect to how we merge Pull Requests and request new releases. This information should be put in the contributing.md file.

The base idea is that anyone can create a PR, but that PR then needs to be accepted by at least one person from another organization actively participating in the collaboration. One important aspect here is that SMALL changes must be quickly reviewed (2-3 **business** days). 
A basic requirement is that AT LEAST one other person from ANOTHER organization must approve the change. That means that even if we all fail our commitment and nobody responds, the PR CANNOT be merged.

Larger or more complex changes or additions shall be discussed in a regular group meeting before being merged.

As soon as someone wants a new release an **issue** should be created.

### PR DESCRIPTION
**Pull request by niclas86**

### CODE REVIEW

**Review by claudiaehr — APPROVED** [review]
Looks good. Small typo "infeature branches" --> "in feature branches"

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches 
+    and shall be code reviewed by at least 2 different companies in the group
+
+# Release:
+    To make a new release, a pull request should be approved by at least 2 
+    different companies in the group. 
+    The release should be taged in git with a release number. 
+    This release number correlate with the version publicized in the IG.
```
Comment (vjernelov) [inline CONTRIBUTING.md]: "Correlate" should be "correlates"

"version publicized" should be "version published"

Comment (niclas86) [reply CONTRIBUTING.md]: Thanks @vjernelov I have now pushed a change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Changes:
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches
```
Comment (vadi2) [inline CONTRIBUTING.md]: I think allowing the 'why' to be stated in the PR itself is also good, this'll save us from creating issues for small things:

```suggestion
    All changes should have an issue in Github or the PR itself that describes well why and what 
    the change aims to achieve. All changes should be made in feature branches 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Changes:
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches 
+    and shall be code reviewed by at least 2 different companies in the group
+
+# Release:
+    To make a new release, a pull request should be approved by at least 2 
+    different companies in the group. 
+    The release should be taged in git with a release number.
```
Comment (vadi2) [inline CONTRIBUTING.md]:
```suggestion
    The release should be tagged in git with a release number. 
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Changes:
+
+    All changes should have an issue in Github that well describes why and what 
+    the change aims to achieve. All changes should be made infeature branches 
+    and shall be code reviewed by at least 2 different companies in the group
```
Comment (vadi2) [inline CONTRIBUTING.md]:
```suggestion
    and shall be code reviewed by at least 2 different companies in the group.
    Small changes must be reviewed quickly (2-3 business days). Larger or more complex
    changes or additions shall be discussed in a regular group meeting before being merged.
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (vjernelov) [reply CONTRIBUTING.md]: I agree and update the branch accordingly.

File: CONTRIBUTING.md (hunk 3 above)
Comment (vjernelov) [reply CONTRIBUTING.md]: Agree and will implement.

File: CONTRIBUTING.md (hunk 4 above)
Comment (vjernelov) [reply CONTRIBUTING.md]: Agree and will implement.

**Pull request merged**